crates/bevy_transform/src/lib.rs

```diff
@@ -90,6 +90,9 @@ pub struct TransformPlugin;
 
 impl Plugin for TransformPlugin {
     fn build(&self, app: &mut App) {
+        #[derive(Debug, Hash, PartialEq, Eq, Clone, SystemSet)]
+        struct Set;
```

Comment:
This ideally needs a more descriptive name. Comments on why it's used would be nice too.